Desktop/exellenteam_python/its_the_way.py

```diff
@@ -0,0 +1,9 @@
+
+import os
```

Comment:
Hi Nechama :)
First, please divide your code into functions (if needed, call them in a main function).
(Remember to document the functions you are about to write).